pkgs/games/anki/default.nix

```diff
@@ -211,6 +277,6 @@ buildPythonApplication rec {
     '';
     license = licenses.agpl3Plus;
     platforms = platforms.mesaPlatforms;
-    maintainers = with maintainers; [ oxij Profpatsch ];
+    maintainers = with maintainers; [ oxij Profpatsch euank ];
```

Comment:
This drv is wildly different to the original to the point where it should be considered separate. Unless the other maintainers agree, you should probably remove them.

@Profpatsch @oxij

Reply:
```suggestion
    maintainers = with maintainers; [ euank ];
```

Reply:
I'll wait to hear from them.
Even though the derivation is different, that doesn't seem a reason to drop em unless they want to be dropped.